core/linker/default.h

```diff
@@ -100,21 +100,35 @@ ENTRY(main_entry)
 #if !defined(STACK_SIZE_NP)
 #if defined (__ARM_FEATURE_CMSE) && (__ARM_FEATURE_CMSE == 3U)
 #define STACK_SIZE_NP 256 /* TODO: choose better default. */
+#define TOTAL_STACK_SIZE_NP STACK_SIZE_NP + STACK_GUARD_BAND
 #else
 #define STACK_SIZE_NP 0
+#define TOTAL_STACK_SIZE_NP 0
 #endif /* defined (__ARM_FEATURE_CMSE) && (__ARM_FEATURE_CMSE == 3U) */
 #endif /* !defined(STACK_SIZE_NP) */
 
+/*
+ *   uVisor SRAM map:
+ * 
+ *                       |-------------UVISOR_SRAM_LENGTH_USED-------------------------|
+ *                       |                                                             |
+ *    |------------------+----------UVISOR_SRAM_LENGTH_PROTECTED-----------------------|
```

Comment:
The length of PROTECTED is not clear by this drawing.

Reply:
add a comment referencing the platform

Reply:
@danny4478 modified - please check again

Reply:
@orenc17A the scheme remains valid for all platforms - the sizes of the regions may vary - 
 `TOTAL_STACK_SIZE_NP`  equals `0` for ARMv7-M